### PR TITLE
Fix Additional Information Text Margin

### PR DESCRIPTION
**Pull request by luacmartins**
### Details
This PR fixes the margin on the additional information page.

### Fixed Issues
$ https://github.com/Expensify/App/issues/5343

### Tests
1. Login to NewDot using an account with a Workspace and without the Expensify Card.
2. Navigate to '/bank-account/contract' or follow [these steps](https://stackoverflow.com/c/expensify/questions/342) to get to the additional information step.
3. Verify that the horizontal margins look like the images below.

### QA Steps
Steps above.

### Tested On

- [X] Web
- [X] Mobile Web
- [X] Desktop
- [X] iOS
- [X] Android

#### Web

web

#### Mobile Web

mweb

#### Desktop

desktop

#### iOS

ios

#### Android

android

### CODE REVIEW

File: src/pages/ReimbursementAccount/BeneficialOwnersStep.js [hunk 1]
```diff
@@ -224,7 +224,7 @@ class BeneficialOwnersStep extends React.Component {
                             )}
                         </View>
                     )}
-                    <Text style={[styles.textStrong, styles.mb5]}>
+                    <Text style={[styles.textStrong, styles.mv5]}>
```
Comment (luacmartins) [inline src/pages/ReimbursementAccount/BeneficialOwnersStep.js:227]: Added a top margin to separate "Agreement" from the top checkboxes and make it look more like @shawnborton's [image](https://github.com/Expensify/App/issues/5343#issuecomment-923042108).

File: src/pages/ReimbursementAccount/BeneficialOwnersStep.js [hunk 2]
```diff
@@ -235,7 +235,7 @@ class BeneficialOwnersStep extends React.Component {
                             <View style={[styles.flexRow]}>
                                 <Text>{this.props.translate('common.iAcceptThe')}</Text>
                                 <TextLink href="https://use.expensify.com/achterms">
-                                    {`${this.props.translate('beneficialOwnersStep.termsAndConditions')}.`}
+                                    {`${this.props.translate('beneficialOwnersStep.termsAndConditions')}`}
```
Comment (luacmartins) [inline src/pages/ReimbursementAccount/BeneficialOwnersStep.js:238]: Removed "." to make it consistent with the other checkbox items.

**Comment by shawnborton** [comment]
I think you can also make the "Agreement" use a normal font weight and not bold.

**Comment by luacmartins** [comment]
Updated!

**Comment by shawnborton** [comment]
Looks good, thanks Carlos!

**Review by Dal-Papa — APPROVED** [review]

**Pull request merged**